download/download.go

```diff
@@ -19,13 +19,14 @@ import (
 // Usage text that will be displayed as command line help text when using the
 // `help download` command
 var Usage = `
-USAGE: %s download [url(s)]
-
-Download: Downloads files from the Sensitive Data Archive (SDA). If a directory is
-          provided (ending with "/"), then the tool will attempt to first
-          download the urls_list.txt file, and then download all files in this
-          list. If file urls are given, they files will be downloaded creating the
-		  same folder structure locally.
+USAGE: %s download (-outdir <dir>) [url(s) | file]
+
+Download: Downloads files from the Sensitive Data Archive (SDA). The files will
+          be downloaded in the current directory, if outdir is not defined.
+		  If a directory is provided (ending with "/"), then the tool will attempt 
+		  to first download the urls_list.txt file, and then download all files in 
+		  this list. If file urls are given, they files will be downloaded creating 
+		  the same folder structure locally.
```

Comment:
There seems to be tabs instead of spaces here, messing up the indentation
```suggestion
          If a directory is provided (ending with "/"), then the tool will attempt 
          to first download the urls_list.txt file, and then download all files in 
          this list. If file urls are given, they files will be downloaded creating 
          the same folder structure locally.
```